main.py

```diff
@@ -1,11 +1,7 @@
 import cv2
-import numpy as np
-from imutils.video import VideoStream
-import argparse
-import imutils
-import time
 from Trackers.centroid_tracker.centroid import Centroid_tracker
 from Detectors.YOLO import yolo
+from Detectors.YOLO.darknet import darknet
```

Comment:
Why there is need to import `darknet` module here? It should be handled in `yolo.py`